packages/db-objects-file/lib/objects/objectsInMemServerRedis.js

```diff
@@ -810,26 +781,27 @@ class ObjectsInMemoryServer extends ObjectsInMemoryFileDB {
     /**
      * Initialize Redis Server
      * @param settings Settings object
-     * @param server Network server to use
+     * @param callback listening/connection callback
      * @private
      */
-    _initRedisServer(settings, server) {
+    _initRedisServer(settings, callback) {
         try {
             if (settings.secure) {
-                throw Error('Secure Redis unsupported');
-            } else {
-                this.server.server = net.createServer();
+                callback && callback(new Error('Secure Redis unsupported for File-DB'));
             }
-            this.server.server.on('error', err => this.log.info(this.namespace + ' ' + (settings.secure ? 'Secure ' : '') + ' Error inMem-objects listening on port ' + (settings.port || 9001)) + ': ' + err);
-            server.server.listen(settings.port || 9001, (settings.host && settings.host !== 'localhost') ? settings.host : ((settings.host === 'localhost') ? '127.0.0.1' : undefined));
+            this.server = net.createServer();
+            this.server.on('error', err =>
+                this.log.info(this.namespace + ' ' + (settings.secure ? 'Secure ' : '') + ' Error inMem-objects listening on port ' + (settings.port || 9001)) + ': ' + err);
+            this.server.on('connection', socket => this._initSocket(socket));
+
+            this.server.listen(
+                settings.port || 9001,
+                (settings.host && settings.host !== 'localhost') ? settings.host : ((settings.host === 'localhost') ? '127.0.0.1' : undefined),
```

Comment:
might wanna turn this check around?
`settings.host === 'localhost' ? '127.0.0.1' : !!settings.host ? settings.host : undefined`

You could even simplify out the last ternary if settings.host is guaranteed to be defined and/or non-empty.

Reply:
chnage it.

BTW: !!settings.host was error in IDE, I removed !!